backend/tracim_backend/app_models/email_validators.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+from marshmallow import ValidationError
+from marshmallow.validate import Email
+
+
+class TracimEmailValidator(Email):
+    """ Special Email Validator who accept email from subdomain"""
+
+    USER_REGEX = re.compile(
+        r"(^[-!#$%&'*+/=?^`{}|~\w]+(\.[-!#$%&'*+/=?^`{}|~\w]+)*\Z"  # dot-atom
+        # quoted-string
+        r'|^"([\001-\010\013\014\016-\037!#-\[\]-\177]' r'|\\[\001-\011\013\014\016-\177])*"\Z)',
+        re.IGNORECASE | re.UNICODE,
+    )
+
+    DOMAIN_REGEX = re.compile(
+        # domain
+        r"(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.?)+" r"(?:[A-Z]{2,6}|[A-Z0-9-]{2,})\Z"
```

Comment:
Simple fix inspirated by this commit: https://github.com/sduthil/marshmallow/commit/126112f3c2f6f0d7ba35e9f960529e8a5d8addd2 .

I'm open to any suggestion to refactor the regex we do use for email. I'm not sure we need such complicated regex.